CHANGELOG.md

```diff
@@ -1,8 +1,12 @@
 # Changelog
 
+## 0.3.3
+
+* Introduce Scope API for Ruby and deprecate `Tracer#start_span` ([#21](https://github.com/opentracing/opentracing-ruby/pull/21))
```

Comment:
Care to explain why deprecate start_span? (maybe also add it to commit message). python api seems to be using also `start_span`